egs/aishell2/asr1/conf/train.yaml

```diff
@@ -0,0 +1,40 @@
+# network architecture
```

Comment:
- Same comment: You can use a symlink for this and specify the actual config in the tuning directory. Please check the other recipe as well.